### PR TITLE
Set up Publishing

### PR DESCRIPTION
**Pull request by LandonSchropp**
* Our gem is now automatically published when pushing to master.
* We updated the `StringLiterals` rule to enforce double quotes.

Paired with @optimumtk

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.6.0
```
Comment (jorge0136) [inline .ruby-version:1]: If we are moving to `2.6.0`, do we need to update our apps to use this version as well? In looking I can see at least `site` is on `2.5.3`.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM ruby:2.6.0
```
Comment (jorge0136) [inline Dockerfile:1]: Same version discrepancy, are we on `2.6.0` or `2.5.3`? Perhaps it does not matter.

File: bin/publish [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```
Comment (jorge0136) [inline bin/publish:1]: Shellcheck? I just ran it locally and nothing was reported back. If we are introducing scripting do we need to pull along the `linting` for it? Probably overkill but I think it's worth asking.

**Review by jorge0136 — CHANGES_REQUESTED** [review]

File: .ruby-version (hunk 1 above)
Comment (LandonSchropp) [reply .ruby-version:1]: Nope, that shouldn't be necessary. 2.6.0 in influencing the version of ruby that's running Rubocop locally, but not the version of Ruby running in dependencies.

File: Dockerfile (hunk 2 above)
Comment (LandonSchropp) [reply Dockerfile:1]: Ditto.

File: bin/publish (hunk 3 above)
Comment (LandonSchropp) [reply bin/publish:1]: I think this problem will be easy to solve if we switch to GitHub actions. However, for this particular use case, I don't feel like it's worth it. I ran it locally while I wrote the script.

File: LICENSE.txt [hunk 4]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2019 Landon Schropp
```
Comment (proneguy) [inline LICENSE.txt]: Copyright 2019 Optimum Energy LLC

**Review by proneguy — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1,27 +1,35 @@
...
+configuration for [Optimum Energy](http://optimumenergyco.com/).
 
 ## Installation
 
-First, add `optimum-energy-rubocopy-config` to your `Gemfile` and run `bundle install`.
+First, add `optimum-energy-rubocop` to your `Gemfile` and run `bundle install`.
 
-Then, create a .rubocop.yml with the following directives:
+First, add the gem to your `Gemfile`. We recommend not locking in the version of the gem because we
+regularly update this repo as we come across new rules.
```
Comment (jonohrt) [inline README.md]: This line is a duplication of the line above it.

**Review by jonohrt — CHANGES_REQUESTED** [review]

File: LICENSE.txt (hunk 4 above)
Comment (LandonSchropp) [reply LICENSE.txt]: Lol, that was autogenerated.

File: README.md (hunk 5 above)
Comment (LandonSchropp) [reply README.md]: Fixed.

File: LICENSE.txt (hunk 4 above)
Comment (proneguy) [reply LICENSE.txt]: Suuuuuuure :P

**Review by proneguy — APPROVED** [review]

**Review by jorge0136 — APPROVED** [review]
Pushed up shellcheck into CI/CD.

**Review by optimumtk — APPROVED** [review]

**Review by BrockBeaudry — APPROVED** [review]

**Review by griesmnr — APPROVED** [review]

**Pull request merged**